docs/reference/training/experiment-config-reference.rst

```diff
@@ -280,6 +280,44 @@ trials will not be restarted and will be marked as errored. An experiment is con
 if at least one of its trials completes without errors. The default value for ``max_restarts`` is
 ``5``.
 
+.. _config-log-policies:
+
+``log_policies``
+================
+
+Optional. The ``log_policies`` parameter is a list of policies that allows certain actions to be
+taken when a trial reports a log that matches the pattern. The pattern is a regex used by the Go
+language. See `this reference <https://github.com/google/re2/wiki/Syntax>`__ for details on the
+regex. The pattern regex is matched line by line against logs reported by the trial. The following
+actions can be taken
+
+-  ``exclude_node``: When a trial fails and restarts according to its ``max_restarts`` policy, if
+   any nodes reported a log that matched the pattern then that node is excluded from scheduling.
+   Only the trial will be prevented from scheduling on a node. This is useful for allowing trials to
```

Comment:
Some copy suggestions:
"if any nodes reported a log that matched the pattern then those nodes are excluded from scheduling for the restart attempts of that trial. Only the failed trial will be prevented from scheduling on those nodes.